src/coredump/coredumpctl.c

```diff
@@ -913,6 +971,10 @@ int main(int argc, char *argv[]) {
                 log_debug("Journal filter: %s", filter);
         }
 
+        units_active = check_units_active();
+        if (units_active < 0)
+                log_warning("Failed to check if any systemd-coredump@.service units are running.");
```

Comment:
hmm, this would mean we log twice about any error. I think it would be better to move this message string into the check_units_active() call, when checken for the errors of sd_bus_call()